certora/specs/Enabled.spec

```
@@ -25,7 +25,7 @@ rule inWithdrawQueueIsEnabledPreservedUpdateWithdrawQueue(env e, uint256 i, uint
     updateWithdrawQueue(e, indexes);
 
     MetaMorphoHarness.Id id = withdrawQueue(i);
-    // Safe require because j is not otherwise constrained.
+    // Safe is require because j is not otherwise constrained.
```

```suggestion
    // Safe require because j is not otherwise constrained.
```

We want to say that the "require" is safe